Kickstarter-iOS/Views/Storyboards/CreditCardCell.xib

```diff
@@ -18,26 +18,27 @@
                 <rect key="frame" x="0.0" y="0.0" width="320" height="65.5"/>
                 <autoresizingMask key="autoresizingMask"/>
                 <subviews>
-                    <stackView opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" alignment="center" spacing="12" translatesAutoresizingMaskIntoConstraints="NO" id="DRF-8a-Nrl">
-                        <rect key="frame" x="6" y="6" width="308" height="53"/>
+                    <stackView opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" alignment="center" spacing="18" translatesAutoresizingMaskIntoConstraints="NO" id="DRF-8a-Nrl">
```

Comment:
We also need to slightly update spacing that was affected by the image view stretching too much